api/permissions_requests_handlers_test.go

```diff
@@ -17,8 +17,7 @@ func TestPermissionsRequestsListHandler(t *testing.T) {
 	saUC := helpers.GetServiceAccountsUseCase(t)
 	sa, err := saUC.CreateKeyPairType("some sa")
 	if err != nil {
-		t.Errorf("Unexpected error %s", err.Error())
-		return
+		t.Fatalf("Unexpected error %s", err.Error())
```

Comment:
nit: Use "%v" and drop the Error() call; it does that by default. Same throughout.